pages/bibliography.md

```diff
@@ -1,11 +1,48 @@
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
+
 # Bibliography
 
----
+1.
+Galka, Max. 2016. "Here's Everyone Who's Immigrated to the U.S. Since 1820." May 3. http://metrocosm.com/animated-immigration-map/.
+
+2.
+Policy, Migration. 2013. "U.S. Immigration Trends." January 23. https://www.migrationpolicy.org/programs/data-hub/us-immigration-trends#lep.
+
+3.
+ProCon.Org. 2017. "Historical Timeline: History of Legal and Illegal Immigration to the United States." January 30. https://immigration.procon.org/view.timeline.php?timelineID=000023.
+
+4.
+Krammer, Jerry. "Immigration: Since the 1965 Immigration Act." Encyclopedia of Religion in America, September 30, 2015. doi:10.4135/9781608712427.n167.https://cis.org/Report/HartCeller-Immigration-Act-1965
+
+5.
+Quinn, Michael. "Why People Migrate: 11 Surprising Reasons." Global Citizen. Accessed July 30, 2019. https://www.globalcitizen.org/en/content/why-people-migrate-11-surprising-reasons/.
+
+6.
+"Migration Data." Migration Data Portal. Accessed July 12, 2019. https://migrationdataportal.org/data?i=stock_abs_&t=2017.
+
+7.
+"Yearbook of Immigration Statistics." Department of Homeland Security. September 13, 2018. Accessed July 12, 2019. https://www.dhs.gov/immigration-statistics/yearbook.
+
+8.
+Cohn, D’Vera, and D’Vera Cohn. "How U.S. Immigration Laws and Rules Have Changed through History." Pew Research Center. September 30, 2015. Accessed July 12, 2019. https://www.pewresearch.org/fact-tank/2015/09/30/how-u-s-immigration-laws-and-rules-have-changed-through-history/.
+
+9.
+Policy, Migration. 2013. "Major U.S. Immigration Laws, 1790 - Present." January 23. https://www.migrationpolicy.org/research/timeline-1790
+
+10.
+Rubin, Ernest. "The Demography of Immigration to the United States." The Annals of the American Academy of Political and Social Science 367 (1966): 15-22. http://www.jstor.org/stable/1034839.
 
-## Project Bibliography
+11.
+Aguila, Emma, Alisher R. Akhmedjonov, Ricardo Basurto-Davila, Krishna B. Kumar, Sarah Kups, and Howard J. Shatz. "Causes of Migration from Mexico to the United States." In United States and Mexico: Ties That Bind, Issues That Divide, 37-48. Santa Monica, CA; Arlington, VA; Pittsburgh, PA: RAND Corporation, 2012. http://www.jstor.org/stable/10.7249/mg985rc.15.
 
-1. List your bibliography items here, whether historical and/or digital in nature, here.
-2. It should be in [Chicago Manual of Style](chicagomanualofstyle.org)
-3. Remember to *italicize* and **bold** as appropriate.
-4. See example below
-5. Clinton, Kevin. *Eleusis, the Inscriptions on Stone: Documents of the Sanctuary of the Two Goddesses and Public Documents of the Deme*. Vol. 1a: Text. Athens: The Archaeological Society at Athens, 2008. 
+12.
+Liskofsky, Sidney. "United States Immigration Policy." The American Jewish Year Book 67 (1966): 164-75. http://www.jstor.org/stable/23604995.
```

Comment:
Overall, good bibliography. Are there any historical monographs on the subject. Often, one or two books tend to have a strong influence on the discussion of an historical topic, even if the more current recent is found in articles.